src/content/docs/en/guides/cms/preprcms.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Prepr CMS & Astro
+description: Add content to your Astro project using Prepr as a CMS
+type: cms
+service: Prepr CMS
+stub: true
+i18nReady: false
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+
+
+[Prepr.io](https://www.prepr.io/), the headless CMS with built-in personalization, that takes your content to a whole new level. Create unique customer experiences and drive conversions like never before.
```

Comment:
The introduction should tell the user what Prepr is, but shouldn't be a marketing piece. Let's try to find something a little more neutral. I've thrown in a suggestion, but feel free to rework it yourself.
```suggestion
[Prepr.io](https://www.prepr.io/) is a schema-based, headless CMS for building websites that serve audience targeted content.
```

Reply:
[Prepr.io](https://www.prepr.io/), is a headless CMS for building marketing sites with Segment targeting.

Reply:
@TheOtterlord I have change the text, do you can review it again?